### PR TITLE
Add CadenzaClient#closeMe function

### PR DESCRIPTION
**Pull request by MrLynx93**

### CODE REVIEW

File: src/docs.md [hunk 1]
```diff
@@ -378,6 +378,15 @@ To invalidate caches and reload:
 cadenzaClient.reload({ invalidateCaches: true });
 ```
 
+### Close itself
+
+<small>API: [CadenzaClient#closeMe](./classes/CadenzaClient.html#reload)</small>
```
Comment (klschm) [inline src/docs.md:383]: The method mentioned here is wrong, it should reference closeMe but references reload

**Review by klschm — CHANGES_REQUESTED** [review]

File: src/cadenza.js [hunk 2]
```diff
@@ -845,6 +846,15 @@ export class CadenzaClient {
     this.#postEvent('reload', { invalidateCaches });
   }
 
+  /**
+   * Sends a message to parent Cadenza window to close the window containing this application
+   */
+  closeMe() {
+    this.#log('CadenzaClient#closeMe');
+    assert(this.#iframe == null, 'Cannot send closeMe to iframe');
```
Comment (jkissel) [inline src/cadenza.js:854]: :+1:

File: src/docs.md [hunk 3]
```diff
@@ -378,6 +378,15 @@ To invalidate caches and reload:
 cadenzaClient.reload({ invalidateCaches: true });
 ```
 
+### Close itself
```
Comment (jkissel) [inline src/docs.md:381]: I think we can be more clear here. Proposal:
> Close the Cadenza Dialog or Window That Embeds the Custom Application

Also note: Headings in technical documentation should be in title case: https://conf.disy.net/display/EHB/AsciiDoc+Documentation+Practices#AsciiDocDocumentationPractices-TitleCase

**Review by jkissel — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## 2.6.0 - 2024-04-12
 ### Added
 - `filter` option for `CadenzaClient#createGeometry`, `CadenzaClient#editGeometry` and `CadenzaClient#selectObjects`
+- `CadenzaClient#closeMe` function
```
[the remark this answers is not in the document]
Comment (MrLynx93) [reply CHANGELOG.md]: right, my mistake, I moved that

**Review by klschm — APPROVED** [review]

**Review by klschm — APPROVED** [review]

File: src/docs.md [hunk 5]
```diff
@@ -378,6 +378,15 @@ To invalidate caches and reload:
 cadenzaClient.reload({ invalidateCaches: true });
 ```
 
+### Close the Cadenza Dialog or Window That Embeds the Custom Application
+
+<small>API: [CadenzaClient#closeMe](./classes/CadenzaClient.html#closeMe)</small>
+
+Sends a message to parent Cadenza window to close the window containing this application.
```
Comment (jkissel) [inline src/docs.md:385]: The Cadenza window is not necessarily the parent window. It can also be the window.opener.

**Review by jkissel — APPROVED** [review]

**Pull request merged**